extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -231,6 +231,11 @@ require([
 			handleClick(event, 'community-page-recent-activity-module');
 		});
 
+		// Track clicks in the Help module
+		$('.helpModule').on('mousedown touchstart', 'a', function (event) {
```

Comment:
https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions